<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWindow</class>
 <widget class="QMainWindow" name="ServerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>440</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Radio Console Server</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grpRadioList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Radio List</string>
    </property>
    <widget class="QListView" name="fRadioListView">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>30</y>
       <width>351</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="fAddRadio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Radio</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fEditRadio">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>320</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit Radio</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fDeleteRadio">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>320</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Radio</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpRadioSettings">
    <property name="geometry">
     <rect>
      <x>389</x>
      <y>10</y>
      <width>401</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Radio Settings</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>29</y>
       <width>381</width>
       <height>281</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="grdRadioSettings">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="horizontalSpacing">
       <number>10</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="4" column="0">
       <widget class="QLabel" name="lblRadioPTT">
        <property name="text">
         <string>PTT Device</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="fEditRadioRxAud">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lblRadioSigMode">
        <property name="text">
         <string>Signalling</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lblRadioSigId">
        <property name="text">
         <string>Signalling ID</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="fEditRadioTxAud">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblRadioDesc">
        <property name="text">
         <string>Description</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="fEditRadioCtrlPort">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="fEditRadioSigMode">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MDC1200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DTMF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Single-tone</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Two-tone</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblRadioTxAud">
        <property name="text">
         <string>Transmit Audio</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lblRadioRxAud">
        <property name="text">
         <string>Receive Audio</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="fEditRadioName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="fEditRadioSigId">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="fEditRadioDesc">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblRadioName">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblRadioCtrlPort">
        <property name="text">
         <string>Control Port</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="fEditRadioPTT">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblRadioCtrlMode">
        <property name="text">
         <string>Control Mode</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="fEditRadioCtrlMode">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="fConnectRadio">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fDisconnectRadio">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>320</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="aMenuSaveConfig"/>
    <addaction name="aMenuLoadConfig"/>
    <addaction name="separator"/>
    <addaction name="aMenuQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="aMenuSaveConfig">
   <property name="text">
    <string>&amp;Save Config</string>
   </property>
  </action>
  <action name="aMenuLoadConfig">
   <property name="text">
    <string>&amp;Open Config</string>
   </property>
  </action>
  <action name="aMenuQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
